engine/classes/Elgg/Debug/Inspector.php

```diff
@@ -167,10 +167,10 @@ public function getActions() {
 	 * @return array [views]
 	 */
 	public function getSimpleCache() {
-		global $CONFIG;
-
+		
+		$simplecache = elgg_extract('simplecache', $this->getViewsData(), []);
 		$tree = array();
-		foreach ($CONFIG->views->simplecache as $view => $foo) {
+		foreach ($simplecache as $view => $foo) {
```

Comment:
Add this in https://github.com/Elgg/Elgg/pull/9668/files#diff-4309aeabc90256cf63cdd7c29cc8deebR176